woocommerce/sift-for-woocommerce
# Add customer payment methods and PPCP chargeback reporting

#### Changes proposed in this Pull Request

* Adds customer payment methods by enumerating over a customer's order and acquiring the payment method for each.
* Removes duplicate payment methods before sending.

#### Testing instructions



* Apply this branch to an environment with multiple payment methods.
* Update the account of a user with multiple payment methods.
* Confirm the payment methods for `$update_account` is correct.
* Try a purchase as a new user and confirm that `$create_account` is correct.

Mentions # 554-gh-automattic/gold
